cmd/fossa/cmd/analyze/analyze.go

```diff
@@ -25,7 +29,8 @@ var Cmd = cli.Command{
 	Action:    Run,
 	ArgsUsage: "MODULE",
 	Flags: flags.WithGlobalFlags(flags.WithAPIFlags(flags.WithModulesFlags([]cli.Flag{
-		cli.BoolFlag{Name: flags.Short(Output), Usage: "show analysis output instead of uploading to FOSSA"},
+		cli.StringFlag{Name: flags.Short(Output), Usage: "send analysis to output file instead of uploading to FOSSA (default: -)"},
+		cli.StringFlag{Name: flags.Short(Template), Usage: "process analysis via template prior to sending it to output"},
```

Comment:
I'd like this usage string to mention that this should be a file name (as opposed to a raw string template).